lib/matplotlib/mathtext.py

```diff
@@ -2726,48 +2759,93 @@ def subsuper(self, s, loc, toks):
             result = Hlist([vlist])
             return [result]
 
-        # Handle regular sub/superscripts
-        shift_up = nucleus.height - SUBDROP * xHeight
-        if self.is_dropsub(nucleus):
-            shift_down = nucleus.depth + SUBDROP * xHeight
+        # We remove kerning for consistency (otherwise it will compute kerning
```

Comment:
Maybe clarfiy: "We remove the kerning on the last character for consistency" (this doesn't remove the inter-character kerning).